### ISSUE TITLE

Is there a way to store action chains for later use?

### ISSUE THREAD

**Issue by Chrysostomus** [uyuni-project/uyuni#8194]
# Question

Is there a way to store action chains for later use?

Use case:
We have do monthly updates to our server fleet. Many server clusters need to be updated and rebooted in certain order and some need remote commands to be run before and after. Action chains would be optimal for this, but creating a new action chain for every update defeats the purpose of automating the operation.

**Comment by mcalmer** [comment]
No. When this feature was developed, it was meant for the traditional stack which had explicit actions like "install that package in that version". You can call this only 1 time.

As of now we did not move this feature into the salt world where you write a profile for a system which you can apply again and again.

But it would be really good to have this.

**Comment by Chrysostomus** [comment]
Thank you for the swift answer! Can I convert this feature into a feature request, or should I open another issue?
